doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
I'm worried because the whitespace confuses git.  Yes, we humans can use brain or tools to figure out what's different, but once such commit mixed with a lot of unrelated changes is put into history, it's really hard to backport, merge with other branches, etc.  Sorry, but it was a big pain to merge conflicts created by some of your past commits again and again: it's really a disaster for integrators and release engineers.  If your editor is stripping all the trailing whitespaces, I would ask you to disable that setting temporarily for DeepDive until we do a full cleanup on every file and add a style check to ban them forever :)  For this one I'll try to merge as it is right after the release, hopefully without much trouble.